.travis.yml

```diff
@@ -0,0 +1,27 @@
+dist: xenial
+language: rust
+sudo: required
+cache: cargo
+env:
+  global:
+    - BUILD=true
+    - CHARM_BUILD_DIR=/tmp/charm-builds
+  matrix:
+    - JUJU_CHANNEL=stable
+    - JUJU_CHANNEL=edge
+matrix:
+  allow_failures:
```

Comment:
Once 2.6 hits stable, we can remove this (2.5.x doesn't support bootstrapping directly onto microk8s).